param_odometerMinNumberMatches: 10
param_odometerRansacIters: 500
param_odometerInlierThreshold: 0.5
param_odometerMotionThreshold: 5.0
param_cameraPitches: [0.0, 0.0, 0.0, 0.0, 0.0]
param_cameraHeights: [1.65, 1.65, 1.65, 1.65, 1.65]